pkg/ingester/ingester.go

```diff
@@ -821,6 +829,31 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	}
 }
 
+func (i *Ingester) updateUserTSDBConfigs() {
+	for _, userID := range i.getTSDBUsers() {
+		userDB := i.getTSDB(userID)
+		if userDB == nil {
+			continue
+		}
+
+		cfg := &config.Config{
+			StorageConfig: config.StorageConfig{
+				ExemplarsConfig: &config.ExemplarsConfig{
+					MaxExemplars: int64(i.limits.MaxExemplars(userID)),
+				},
+			},
+		}
+
+		level.Info(logutil.WithUserID(userID, i.logger)).Log("meg", "updating max exemplars configuration.")
```

Comment:
typo s/meg/msg/g. I think we can remove this log line since it is very frequent and verbose.

Reply:
Makes sense. I was using it for my testing and forgot to remove it.